src/components/CatalogPreview/__test__/ObsoleteWarning.test.js

```diff
@@ -1,9 +1,11 @@
 import React from 'react'
 import { shallow } from 'enzyme'
-import ObsoleteWarning from '../ObsoleteWarning'
+import {ObsoleteWarning} from '../ObsoleteWarning'
+
+const t = k => k
```

Comment:
Can you give it a better nam? Like `translateMock`, so we can understand easily.